whisp_feature_collection.ipynb

```diff
@@ -228,17 +232,31 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "output_file=out_directory/'whisp_output_table_w_risk.csv' # edit as required\n",
+    "# Create a copy of the DataFrame for export purposes\n",
+    "df_for_export = df_w_risk.copy()\n",
+    "\n",
```

Comment:
thanks. useful addition for notebooks. I wonder if there is a cleaner way as NAs...say adding to the stats automatically like the plot_id is (or when it works it should be)? 
Until then this will sort the issue. Only think is can we slide it into the same position (order) as if you generated them? Thinking consistency.